ambry-router/src/main/java/com.github.ambry.router/PutOperation.java

```diff
@@ -857,6 +941,7 @@ void checkAndMaybeComplete() {
             prepareForSending();
           } else {
             // this chunk could not be successfully put. The whole operation has to fail.
+            logger.error("Elapsed slipped puts. Failing the operation for {}", getChunkBlobId());
```

Comment:
elapsed? Not sure I understand this log statement.

Reply:
I mean, we exhausted all of slipped puts. "elapsed" seems okay to be. if not, do you have any suggestions on how to rephrase.

Reply:
"Slipped puts failed" is probably better